src/core/app_config/index.ts

```diff
@@ -52,7 +29,7 @@ export default (() => {
     layersByDefault: konturAppConfig.LAYERS_BY_DEFAULT,
     featuresByDefault: konturAppConfig.FEATURES_BY_DEFAULT,
     defaultFeed: konturAppConfig.DEFAULT_FEED,
-    defaultFeedObject: getDefaultFeedObject(konturAppConfig.DEFAULT_FEED),
+    defaultFeedObject: DEFAULT_FEED_OBJECT, // translation should occur later after i18n init, getDefaultFeedObject(konturAppConfig.DEFAULT_FEED),
```

Comment:
found that it could be done this way
 ```
 i18n
  .use(LanguageDetector)
  .use(initReactI18next)
  .init(
    {
      fallbackLng: 'en',
      debug: false,
      defaultNS: 'common',
      interpolation: {
        escapeValue: false,
      },
      resources: {
        en,
        es,
        ar,
      },
    },
    (_, t) => {
      DEFAULT_FEED_OBJECT.name = t('logout');
    },
    ```
it's clumsy, but maybe for this unusual case it's ok ? just because I'm not sure it you wan't to await i18n before you get config
BTW do you have an idea of how to implement it later?

Reply:
As I known @vkozio done it that way for solve cycled deps. 
I have an ideas how to solve it in general, it out of scope of this task